src/main/java/com/bloxbean/cardano/client/common/model/Networks.java

```diff
@@ -10,4 +10,14 @@ public static Network testnet() {
         Network testnet = new Network(0b0000, 1097911063);
         return testnet;
     }
+
```

Comment:
This looks good to me. You may want to change the local variable name in prepod & preview methods to prepod/preview instead of testnet.